src/measure_theory/ae_eq_fun.lean

```diff
@@ -286,6 +461,19 @@ instance : emetric_space (α →ₘ γ) :=
 
 lemma edist_mk_mk {f g : α → γ} (hf hg) : edist (mk f hf) (mk g hg) = ∫⁻ x, edist (f x) (g x) := rfl
 
+lemma edist_to_fun (f g : α →ₘ γ) : edist f g = ∫⁻ x, edist (f.to_fun x) (g.to_fun x) :=
+by conv_lhs { rw [self_eq_mk f, self_eq_mk g, edist_mk_mk] }
+
+lemma edist_zero_to_fun [has_zero γ] (f : α →ₘ γ) : edist f 0 = ∫⁻ x, edist (f.to_fun x) 0 :=
+begin
+  rw edist_to_fun,
+  apply lintegral_congr_ae,
+  have : ∀ₘ a:α, (0 : α →ₘ γ).to_fun a = 0 := zero_to_fun,
+  filter_upwards [this],
+  assume a, simp only [mem_set_of_eq],
+  assume h, rw h
```

Comment:
Do these two lines correspond to one or two goals? If they correspond to two goals, each goal proof should be enclosed with `{ ... }`. If they correspond to one goal, each statement should be on its own line.